test/libsolidity/syntaxTests/functionCalls/calloptions_repeated.sol

```diff
@@ -4,7 +4,7 @@ contract C {
         this.foo{value:2, gas: 5}{gas:2};
         (this.foo{value:2, gas: 5}){gas:2};
         this.foo{value:2, gas: 5}{value:6};
-        this.foo.value(4){value:2, gas: 5};
```

Comment:
This one should actually probably stay - it's good to check that we correctly detect repeated use with mixed syntax.

Reply:
changed back